rsa-cryptosystem-signature/index07.js

```diff
@@ -0,0 +1,17 @@
+function hashTheMessage(message) {
+  let hashValue = 0;
+  for (let i = 0, msgLength = message.length; i < msgLength; ++i) {
+    hashValue += message.charCodeAt(i);
+  }
+  return hashValue;
+}
+
+function generatePrivateKey() {}
+
+function generatePublicKey() {}
+
+/*
+For different private keys, there's a different public key.
```

Comment:
Each private key is paired with a public key.

Create a variable `publicKey` and set it to 0.